frontend/packages/noobaa-storage-plugin/src/components/resource-providers-card/resource-providers-card.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardHelp,
+  DashboardCardTitle,
+} from '@console/internal/components/dashboard/dashboard-card';
+import {
+  DashboardItemProps,
+  withDashboardResources,
+} from '@console/internal/components/dashboards-page/with-dashboard-resources';
+import { ResourceProviderItems, ProviderType } from './resource-providers-card-body';
+import './resource-providers-card.scss';
+
+const RESOURCE_PROVIDERS_QUERY = {
+  PROVIDERS_TYPES: ' NooBaa_cloud_types',
+  UNHEALTHY_PROVIDERS_TYPES: 'NooBaa_unhealthy_cloud_types',
+};
+
+const getProviderType = (provider) => _.get(provider, 'metric.type', null);
+const getProviderCount = (provider) => Number(_.get(provider, 'value[1]', null));
+
+const createProvidersList = (data) => {
+  const providers = _.get(data, 'data.result', null);
+  const providersList: ProviderType = {};
+  if (_.isNil(providers)) return null;
+  providers.forEach((provider) => {
+    providersList[getProviderType(provider)] = getProviderCount(provider);
+  });
+  return providersList;
+};
+
+const ResourceProviders: React.FC<DashboardItemProps> = ({
+  watchPrometheus,
+  stopWatchPrometheusQuery,
+  prometheusResults,
+}) => {
+  React.useEffect(() => {
+    watchPrometheus(RESOURCE_PROVIDERS_QUERY.PROVIDERS_TYPES);
+    watchPrometheus(RESOURCE_PROVIDERS_QUERY.UNHEALTHY_PROVIDERS_TYPES);
+    return () => {
+      stopWatchPrometheusQuery(RESOURCE_PROVIDERS_QUERY.PROVIDERS_TYPES);
+      stopWatchPrometheusQuery(RESOURCE_PROVIDERS_QUERY.UNHEALTHY_PROVIDERS_TYPES);
+    };
+  }, [watchPrometheus, stopWatchPrometheusQuery]);
+
+  const providersTypesQueryResult = prometheusResults.getIn([
+    RESOURCE_PROVIDERS_QUERY.PROVIDERS_TYPES,
+    'result',
+  ]);
+  const unhealthyProvidersTypesQueryResult = prometheusResults.getIn([
+    RESOURCE_PROVIDERS_QUERY.UNHEALTHY_PROVIDERS_TYPES,
+    'result',
+  ]);
+
+  const providersTypes = createProvidersList(providersTypesQueryResult);
+  const unhealthyProvidersTypes = createProvidersList(unhealthyProvidersTypesQueryResult);
+
+  return (
+    <DashboardCard>
+      <DashboardCardHeader>
+        <DashboardCardTitle>Resource Providers</DashboardCardTitle>
+        <DashboardCardHelp>
+          A list of all MCG (Multi-cloud gateway) resources that are currently in use. Those
+          resources are used to store data according to the buckets policies and can be a
+          cloud-based resource or a bare metal resource.
+        </DashboardCardHelp>
+      </DashboardCardHeader>
+      <DashboardCardBody>
+        <ResourceProviderItems
```

Comment:
can you do this in a similar way to other cards where we have `Body` and `Items` ? it would look something like

```
<DashboardCardBody>
   <ResourceProviderBody isLoading=... hasProviders=...>
    {providerTypes.map(provider => <ResourceProviderItem ... />}
  </ResourceProviderBody>
</DashboardCardBody>
```